### PR TITLE
Maps API: Map with Icons

### PR DESCRIPTION
**Pull request by tgomezzzz**
In this PR, I changed the Bucket List tab to display a map that shows my favorite places. Using the Google Maps JS API, a map is embedded in the tab that shows some of my favorite places as icons. When clicked, each icon shows more information about the place in a separate DOM next to the DOM with the map. Additionally, if the user clicks anywhere on the map other than on an icon, a DOM appears where they can enter more information about the place. I subsequent PRs, I plan to use this window to allow users to enter their own favorite places, which will then be saved to the Datastore.

### CODE REVIEW

**Comment by vidhyal** [comment]
Please resolve conflicts.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -211,6 +207,103 @@ <h3>College - present</h3>
             computer science remains my strongest interest, and I'm overjoyed to be able to pursue the subject I'm most passionate about.
         </p>
       </div>
+
+      <div id="hood-river" class="more-info">
+        <h2>Hood River, Oregon</h2>
+        <img src="/images/hood-river.jpg" alt="Hood River, Oregon" style="width: 85%">
```
Comment (vidhyal) [inline portfolio/src/main/webapp/index.html:213]: optional: style using a class.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -211,6 +207,103 @@ <h3>College - present</h3>
...
+      <div id="create-marker" class="more-info">
+        <h2>Add a New Location</h2>
+        <form action="/data" method="POST">
+          <label for="creator-namee">Your name:</label></br>
+          <input type="text" id="creator-name" name="creator-name"></br></br>
+          <label for="location-name">Location name:</label></br>
+          <input type="text" id="location-name" name="location-name"></br></br>
+          <label for="location-info">Tell us about this place:</label></br>
+          <input type="text" id="location-info" name="location-info"></br></br>
+          <input type="submit" id="submit"/></br></br>
```
Comment (vidhyal) [inline portfolio/src/main/webapp/index.html:304]: Sorry, it is not clear what happens when user does submit.

**Review by adamtcohen — APPROVED** [review]
As discussed only did brief for review for critical issues.

**Pull request merged**